dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/index_stats.jsp

```diff
@@ -161,11 +155,11 @@ Map<String,ClusterIndexHealth> map = esapi.getClusterHealth();
 					<td><%=UtilMethods.webifyString(myDate) %></td>
 
 					<td align="center">
-						<%=(status !=null && status.getTotal() !=null && status.getTotal().getDocs() != null) ? status.getTotal().getDocs().getCount(): "n/a"%>
+						<%=status !=null ? status.getDocumentCount() : "n/a"%>
 					</td>
 					<td align="center"><%=(health !=null) ? health.getNumberOfShards() : "n/a"%></td>
 					<td align="center"><%=(health !=null) ? health.getNumberOfReplicas(): "n/a"%></td>
-					<td align="center"><%=(status !=null && status.getTotal() !=null && status.getTotal().getStore() !=null) ? status.getTotal().getStore().size(): "n/a"%></td>
+					<td align="center"><%=status !=null ? status.getSize(): "n/a"%></td>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)